This workaround was put in the driver to easily build
in STA configurations without mac80211 modifications, but
is inappropriate in OpenWrt (when we have the necessary
patches in mac80211). See SW-14709.

diff --git a/mac.h b/mac.h
index 4f2e451b..943c78ea 100644
--- a/mac.h
+++ b/mac.h
@@ -14,13 +14,6 @@
 /* The maximum number of frames to send after a DTIM to firmware */
 #define MORSE_MAX_MC_FRAMES_AFTER_DTIM (10)
 
-/* Re-Define the IGNORE channel flag, if not defined by the cfg80211 patch.
- * The flag won't be used by MM81xx.
- */
-#if defined(__x86_64__)
-#define IEEE80211_CHAN_IGNORE	IEEE80211_CHAN_DISABLED
-#endif
-
 /**
  * struct morse_queue_params - QoS parameters
  *
